src/domain/community/user-group/user-group.dto.remove.member.ts

```diff
@@ -0,0 +1,10 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class RemoveUserGroupMemberInput {
+  @Field({ nullable: false })
+  parentID!: number;
```

Comment:
This is really super generic - needs to be discussed

Reply:
Ok